build.gradle

```diff
@@ -59,75 +62,11 @@ subprojects {
         }
     }
     check.dependsOn jacocoTestReport
-
-//    signing {
-//        sign publishing.publications
-//    }
-
-    publishing {
-        publications {
-            lib(MavenPublication) {
-                pom {
-                    url = 'http://pravega.io'
-                    scm {
-                        url = 'https://github.com/pravega/pravega-keycloak/tree/master'
-                        connection = 'scm:git:git://github.com/pravega/pravega-keycloak.git'
-                        developerConnection = 'scm:git:https://github.com/pravega/pravega-keycloak.git'
-                    }
-                    licenses {
-                        license {
-                            name = 'The Apache License, Version 2.0'
-                            url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-                        }
-                    }
-                }
-            }
-        }
-    }
-
-    artifactory {
```

Comment:
Is the idea that we stop publishing to oss.jfrog.org? I don't see this being moved somewhere else, so I'm interpreting that this is what we are trying to achieve.

Also, have we been publishing snapshots?

Reply:
The `maven-publish` plugin is used for publishing to oss.jfrog too. 

https://github.com/pravega/pravega-keycloak/blob/5badd12255c3373823c7757890aa68ec6e62c3e5/gradle/maven-publish.gradle#L79